submissions/fiyingabriel/HTML/prd for html task.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE HTML>
+<html>
+<title>
+Product Page 	
+</title>
+<head>
+	<meta charset="utf-8">
+</head>
+  
+<body>
+	<p> <img src="tte.jpg" height="200" width="2500"> <img src="OIP.jpg" height="100"width="300"></p>
+	<h1>Apple iPhone X - 64GB</h1>
+	<div>
+    <p><img src="149543_1549727850.jpg" height="500px" width="500px"></p>
+    <h1 align="left">N250,000</h1>
+    <img src ="BuyNowButton.png"height = "100" width="200">   
```

Comment:
You are meant to use only 2 product images. Check the criteria here.
https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/issues/7
You should the images that you are not using.

Reply:
I only used 2 product images

Reply:
I dont understand

Reply:
this is the other half of the page
![Screenshot (14)](https://user-images.githubusercontent.com/67837755/87441383-f44e1b00-c5ea-11ea-9fcb-dc7e17e7c42b.png)

Reply:
You should delete the images you're not using that you've added .

Reply:
Im using all of them

Reply:
Which one should i remove